pkgs/applications/virtualization/libnvidia-container/default.nix

```diff
@@ -7,6 +7,8 @@
 , libseccomp
 , rpcsvc-proto
 , libtirpc
+, makeWrapper
+
```

Comment:
```suggestion
, makeWrapper
```

Reply:
It doesn't look like I can commit this change because GH's UI thinks there were no changes. What change were you suggesting here?

Reply:
Oh, maybe it's just removing the line break here.

Reply:
Yeah... the github suggestions feature does not like blank linces.